components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/LightWeightKeyManagerImpl.java

```diff
@@ -0,0 +1,469 @@
+package org.wso2.carbon.apimgt.core.impl;
```

Comment:
Shall we change the name of the class? I guess we don't need to explicitly say about lightweight key manager.